### PR TITLE
Improved generator objectives for GANs

### PR DESCRIPTION
**Pull request by mikkel**
Improved generator objectives for GANs

https://arxiv.org/pdf/1612.02780v1.pdf

```json
"g_loss_type": "standard"
"g_loss_type": "rkl"
"g_loss_type": "kl"
"g_loss_type": "alpha"
"alpha": 0 to 1
```

### CODE REVIEW

File: hypergan/losses/f_divergence_loss.py [hunk 1]
```diff
@@ -68,4 +68,14 @@ def _create(self, d_real, d_fake):
         d_loss = -gfx+conjugate
         g_loss = -conjugate
 
+        if config.g_loss_type == 'standard':
+            g_loss = -conjugate
+        elif config.g_loss_type == 'rkl':
```
Comment (martyn) [inline hypergan/losses/f_divergence_loss.py]: s/rkl/reverse_kl to match other label

**Comment by mikkel** [comment]
Merged manually.
